### ISSUE TITLE
Add connection (power / console / interface) to api 

### ISSUE THREAD

**Issue by powerscooter** [netbox-community/netbox#9684]
### NetBox version

v3.2.3

### Feature type

New functionality

### Proposed functionality

<img width="250" alt="image" src="https://user-images.githubusercontent.com/18429672/177936998-c2e13920-ca60-4b17-8fee-2dab1dc0dfa8.png">

Hello, is it possible to make the connections, shown in the above screenshot, directly queryable via API?
So that i get a list only from the power, consoles or interface connections?

### Use case

We use an application interface where we retrieve the data from Netbox via API. Since the list of cables is now very long, the query takes about 6-8 seconds. 
By using filters, we were able to shorten this to less than 2 seconds.

If we could pull the lists directly one by one, there would be no need to build tedious filters here.

**Comment by jeremystretch** [comment]
These endpoints were intentionally removed under #5223 as they are no longer needed. Additionally, cabling was overhauled in NetBox v3.3 and you may experience better performance on a newer release. If this continues to be a concern, please consider submitting a feature request to explore how the performance of existing API endpoints might be improved.
